### DISCUSSION TITLE
More options for handling audio files

### DISCUSSION THREAD

**Discussion by QuetzalcoutlDev** [4ian/GDevelop#5535] [Enhancement ideas (and their technical discussions)]
Something I just realized while testing GDevelop, is that when it comes to playing audio (music or sound), it allows you to play audio in a loop but it doesn't allow you to set the audio loop points, get the duration of the audio or play the audio from a certain point.

I think it would be very useful if GDevelop could allow you to set the loop points, get the duration or play the audio from a certain point, but the serious question is, is it possible that this can be implemented in GDevelop?

**Comment by 4ian** [comment]
Should be doable as long as it's supported by https://github.com/goldfire/howler.js#documentation - and by extension by the HTML5 audio API/WebAudio API.

**Comment by QuetzalcoutlDev** [comment]
Well that's true
